xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -246,6 +246,14 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
 ##
 ##
 ##
+#macro(displayCommentButton)
+<button type="button" id="OpenCommentForm" class="btn btn-default">
```

Comment:
IIRC the recommended practice, as I'm not sure we have it documented somewhere, is to use `camelCase` format for class and ids naming.

Reply:
Fixed

Reply:
@acotiuga we recommend using `camelCase` for class names? not snakeCase? i.e. `myClassName` and not `my-class-name`?

Reply:
I know the `xHint`, `xForm` classes, I also found https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-annotation/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/Tab.xml#L46 and there are many more.
I personally use `camelCase` in classes and ids, as I also do with Velocity, JavaScript and Java variables.
The snake-case is rather from the old code and most of the time it is demanded by third party libraries (bootstrap). 
Maybe we can get another opinion, @gdelhumeau could know something about this.

Reply:
I'd also prefer camelcase for html attribute values but we may have used snakecase more. We don't have any convention ATM for this AFAIK. Would be good to decide it with a proposal on the forum.

Reply:
Note that we cannot change existing ids and classes as it would break backward compat.

Reply:
See also https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/

Reply:
And https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/#HIssues ;)

Reply:
Unless I missed something, I hope I'm conforming to the code style linked above.